tests/integration-tests/tests/update/test_update.py

```diff
@@ -89,22 +89,25 @@ def test_update_slurm(region, pcluster_config_reader, clusters_factory, test_dat
     _assert_launch_templates_config(queues_config=initial_queues_config, cluster_name=cluster.name, region=region)
 
     # Submit a job in order to verify that jobs are not affected by an update of the queue size
-    result = slurm_commands.submit_command("sleep infinity", constraint="static")
+    result = slurm_commands.submit_command("sleep infinity", constraint="static&c5.xlarge")
```

Comment:
not sure I understand why this behavior changed in pcluster3

Reply:
Neither do I. I will look into it if necessary

Reply:
So the problem was that after we removed the initial_count and used the static min count to do the test.
And the output of sinfo is:
```
PARTITION AVAIL TIMELIMIT NODES STATE NODELIST
queue1*   up  infinite   10 idle~ queue1-dy-c5xlarge-1,queue1-dy-t2micro-[1-9]
queue1*   up  infinite   2  idle queue1-st-c5xlarge-1,queue1-st-t2micro-1
queue2    up  infinite   10 idle~ queue2-dy-c5n18xlarge-[1-10]
```
And t2micro is assigned a job first
Previously, the t2.micro was a dynamic nodes